share/github-backup-utils/ghe-backup-storage

```diff
@@ -37,6 +37,8 @@ user="${host%@*}"
 hostnames=$host
 ssh_config_file_opt=
 tempdir=$(mktemp -d -t backup-utils-restore-XXXXXX)
```

Comment:
As mentioned, the `tempdir` needs to be created on the remote appliance too, as per https://github.com/github/backup-utils/blob/legacy/share/github-backup-utils/ghe-backup-alambic-cluster-ng#L40.